code/style.css

```diff
@@ -138,3 +160,5 @@ button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+
```

Comment:
Great job Rosanna, it's a very nice design on the bot and all the necessary functions are there!
Your code is very structured and easy to follow with all the comments, which I really like. One can tell that you have really thought about a lot of things like people ordering from their phone (with the uppercase letters starting) to have the bot restart when someone answers that they are not satisfied with their order etc. You have completed the requirements for both BLUE and RED level. Please let me know if you have any comments or questions on my review and feedback.

Reply:
Thank you so much for your feed-back Jenny! 👍